modules/ROOT/pages/mergetags.adoc

```diff
@@ -0,0 +1,68 @@
+= Merge Tags plugin
+:navtitle: Merge Tags
+:description: Enables Merge Tags functionality in the {productname} editor
+:keywords: merge, tags, tag, mergetags, mergetag, mergetags_list, mergetags_prefix, mergetags_suffix, mail merge, personalization token
+:pluginname: Merge Tags
+:plugincode: mergetags
+:pluginminimumplan: tiertwo
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows the user to insert a merge tag (also known as a personalization token, or a mail merge field).
+
+Merge tags can be inserted as-you-type; selected from a drop-down list when a specified prefix is typed; or selected and inserted from the searchable Merge tags toolbar menu button.
```

Comment:
```suggestion
Merge tags can be inserted as-you-type, selected from a drop-down list when a specified prefix is typed; or selected and inserted from the searchable Merge tags toolbar menu button.
```
Just clarifying that there are only two methods of inserting a merge tag. First is as you type by selecting from a drop down when a specified prefix is typed, and second is via the toolbar menu button. This use of semi-colons implies three distinct methods.

Reply:
Change made locally and pushed, because I made even further edits to the ones suggested above.